plugins/scaffolder/src/api.ts

```diff
@@ -49,17 +51,18 @@ export class ScaffolderApi {
     });
 
     if (response.status !== 201) {
-      throw new Error(await response.text());
+      const status = `${response.status} ${response.statusText}`;
+      const body = await response.text();
```

Comment:
I'd slap a `.trim()` on there

Reply:
y tho, it's being slapped into the DOM right :)

Reply:
Because it's being templated into the error message

Reply:
well ok then, added